integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLoggingFluentd.java

```diff
@@ -174,32 +169,32 @@ public static void init(@Namespaces(3) List<String> namespaces) {
     // install and verify Kibana
     logger.info("install and verify Kibana");
     kibanaParams = assertDoesNotThrow(() -> installAndVerifyKibana(elasticSearchNs),
-        String.format("Failed to install Kibana"));
+            String.format("Failed to install Kibana"));
```

Comment:
There are a lot of changes in this file that are just spacing. Probably because the default formatting in your IDE is creating unnecessarily large indents. Can you please try to match what was here? I think it's generally two spaces.

Reply:
Sorry, just moved to a new env, IDE has not been fully configured, will do.